medmodels/data_synthesis/mtgan/model/critic/critic.pyi

```diff
@@ -0,0 +1,41 @@
+import torch
+from torch import nn
+
+from medmodels.data_synthesis.mtgan.model.gan import TrainingHyperparametersTotal
+from medmodels.data_synthesis.mtgan.model.generator.generator import Generator
+from medmodels.data_synthesis.mtgan.model.loaders import MTGANDataLoader
+from medmodels.data_synthesis.mtgan.model.real_gru.real_gru import RealGRU
+
+class Critic(nn.Module):
+    real_gru: RealGRU
+    device: torch.device
+
+    number_codes: int
+    critic_hidden_dimension: int
+    generator_hidden_dimension: int
+    critic_iterations: int
+    batch_size: int
+
+    optimizer: torch.optim.optimizer.Optimizer
+    scheduler: torch.optim.lr_scheduler.LRScheduler
+    loss_function: nn.Module
+    linear_layer: nn.Sequential
+
+    def __init__(
+        self,
+        real_gru: RealGRU,
+        number_codes: int,
```

Comment:
can this be renamed to something more expressive or "at least" `number_of_codes` or sth. Feels like a little itch in my brain :D

Reply:
also the other `number_...` in the other files

Reply:
but don't force it. if you can't find anything better, I'll just have to learn to live with that itch in my brain